app/assets/javascripts/variables.js.erb

```diff
@@ -1,2 +1,2 @@
 <% environment.context_class.instance_eval { include Rails.application.routes.url_helpers } %>
-var new_user = "<%= "#{Rails.application.config.action_controller.relative_url_root.to_s + quick_new_users_path}" + " #content" %>";
+var new_user = "<%= "#{Rails.application.config.action_controller.relative_url_root.to_s + quick_new_users_path}" + " #content *" %>";
```

Comment:
I was having a problem with the quick new user modal, where it was loading in the `#content` div in its entirety from `layouts/_content.html.erb` inside of the modal, which was screwing up the modal display. So `#content *` is a standard jQuery / CSS selector that just selects all the elements inside of that div, instead of the div itself.

See http://api.jquery.com/load/#loading-page-fragments